components/icon-button/style.scss

```diff
@@ -15,6 +15,13 @@
 		flex: 0 0 auto;
 	}
 
+	// Ensure that even SVG icons that don't include the .dashicon class are colored
+	.dashicon,
```

Comment:
This line specifically may be redundant if it's already expected to be styled from `components/dashicon/style.scss`.